client/web/src/components/fuzzyFinder/FuzzyModal.module.scss

```
@@ -12,40 +12,78 @@
 }
 
 .content {
+    display: flex;
+    align-items: stretch;
+    flex-direction: column;
+    height: 80vh;
     width: 80vw;
-    background-color: var(--color-bg-2);
+    padding: 1rem;
```

Just checked Figma again, this should be:

```suggestion
    padding: 1.5rem;
```